akka-stream-tests/src/test/scala/akka/stream/scaladsl/FramingSpec.scala

```diff
@@ -5,16 +5,15 @@ package akka.stream.scaladsl
 
 import java.nio.ByteOrder
 import java.util.concurrent.ThreadLocalRandom
-import java.util.concurrent.atomic.AtomicLong
 
+import akka.stream._
 import akka.stream.scaladsl.Framing.FramingException
 import akka.stream.stage.{ GraphStage, _ }
 import akka.stream.testkit.{ StreamSpec, TestPublisher, TestSubscriber }
-import akka.stream._
 import akka.util.{ ByteString, ByteStringBuilder }
+import org.scalatest.concurrent.PatienceConfiguration.Timeout
```

Comment:
How is this configured and what are the default values?
Do I read it correctly that the default timeout is 150 ms, and the retry interval is 15 ms?

I'm not sure if it's good to start mixing this with the testkit way of doing it. Apart from that it's cool to use latest scalatest features, what is the advantage of using this?